the-right-mindset.md

```diff
@@ -1,12 +1,12 @@
 #### The Growth Mindset
 
-Dr. Carol Dweck's research on learning revealed two mindsets, the growth mindset and the fixed mindset. She found that subjects with the growth mindset responded more positively to difficult challenges and understood that although they might not able to solve some difficult problems yet, they could develop the ability to solve them later. The key to this is their "not yet" attitude. When they encounter errors, they had a higher measured brain activity than their fixed mindset counterparts. The subjects with the growth mindset frequently learned from their errors and corrected them.
+Dr. Carol Dweck's research on learning revealed two mindsets - the growth mindset and the fixed mindset. Dweck found that subjects with the growth mindset responded more positively to difficult challenges and understood that, although they might not able to solve them yet, they could develop the ability to solve difficult problems later. The key to this is the "not yet" attitude. When supbjects encountered errors, they had a higher measured brain activity than their fixed mindset counterparts. The subjects with the growth mindset frequently learned from their errors and corrected them.
 
 Subjects with a fixed mindset did not respond well to difficult challenges and often felt like a failure for not being able to solve a problem. Dr. Dweck described them as feeling "crushed by the tyranny of now" and asserted that they often avoid difficult problems.
 
-I see characteristics of both mindsets in myself tough I primarily view myself as having a growth mindset. While it does bother me when I can't learn a concept as quickly as i would like, i understand that my abilities can be developed further. I try my best to learn from errors and not get discouraged.
+I see characteristics of both mindsets in myself. I primarily view myself as having a growth mindset but it does sometimes bother me when I can't learn a concept as quickly as i would like. I do understand that my abilities can be developed further and try my best to learn from errors and not get discouraged.
 
-I plan to use the concept of the growth mindset to help me succeed in my Iron Yard course. I expect there to be many occasions in which I don't understand the concepts at first. Knowing that I can work at developing those skills should keep me on track throughout the program.
+I plan to use the concept of the growth mindset to help me succeed in my Iron Yard course. I expect there to be many occasions in which I don't understand the concepts at first but knowing that I can work at developing those skills should keep me on track throughout the program.
```

Comment:
Thank you for working hard on this assignment! I know you said writing isn't your favorite thing to do, but you did great. :)